Automattic/newspack-plugin
# feat(block-editor): utility to relink the editor close button to a wizard screen

### All Submissions:






### Changes proposed in this Pull Request:



Adds a small utility to relink the full-screen close button in the block editor to a corresponding Newspack wizard page, for Newspack CPTs that have one.

<img width="1374" alt="T6X6AN" src="https://user-images.githubusercontent.com/2230142/138364907-c8556fe2-f812-4c45-b5cb-60a6db708501.png">

Closes #1205.

### How to test the changes in this Pull Request:

1. On `master`, edit a Campaigns prompt. Observe that the full-screen close button (see screenshot above) links to the standard WP admin post list UI for Prompts, which is normally hidden when the Newspack plugin is active.
2. Check out this branch, edit the prompt. Now confirm that the close button links to the Campaigns wizard page instead.
3. Temporarily deactivate the Newspack Plugin and edit the prompt again. Confirm that the close button once again links to the WP admin post list for Prompts.
4. Reactivate the Newspack Plugin. Edit another standard WP post type (like a post or page). Confirm that the close button links to the WP admin post list for that post type, as usual.

### Other information:
